woudc_data_registry/models.py

```diff
@@ -348,6 +353,7 @@ def __geo_interface__(self):
                 'station_name': self.station.station_name.name,
                 'data_class': self.dataset.data_class,
                 'dataset': self.dataset_id,
+                'deployment_id': self.deployment_id,
```

Comment:
The dictionary you return from this method should exactly match up with what's defined in the ElasticSearch mapping in `search.py`: that means there should be a `contributor_name` here instead of `deployment_id`.